draft-tigress-general-requirements.md

```diff
@@ -182,9 +182,11 @@ sequenceDiagram
 - (Req-Privacy) Transport protocol used to transfer provisioning information ( e.g. secure E2E transfer protocol or intermediary server) shall prevent from correlating users between exchanges or create a social graph of users involved into transfer. Intermediary server shall not be an arbiter of identity. User identities shall not be collected, stored and used for purpose other then the credential transfer itself.
 - (Req-Connectivity) Sender and Receiver shall be allowed to be online at different times. Sender and Receiver shall not need to be online at the same time. This requirement allows devices to connect to network to only exchange the portion of information required during the transfer, allowing them upload or download data in turns to network servers.
 - (Req-RoundTrips) Solution shall allow for multiple data exchanges between sender and receiver devices in the process of credential transfer. This requirement shall alighn with (Req-Connectivity) above.
+- (Req-Speed) When both Sender and Receiver are online at the same time they should be able to quickly and efficiently transfer data.
```

Comment:
I am not quite certain that we need this. Previously folks argued these are implementation details, not requirements (per your explanation this refers to push notification to devices )

Reply:
I think the push notifications as a solution is an implementation detail, but the requirement for it to be fast could be a general requirement.